### ISSUE TITLE
Error when starting kernel: "The following argument was not expected" with custom interpreter Python 3.10 Windows standalone installer for all users installation

### ISSUE THREAD

**Issue by Pigankle** [spyder-ide/spyder#19731]
### What steps will reproduce the problem?

For my installation, when I open Spyder I get a red box with "Error when starting kernel: "The following argument was not expected: path/to/installation.  This is akin to error #19298.  Sometimes if I restart the kernel it will begin working (although it still gives me a plaintext version of the same error.  Sometimes, such as this one, it will not allow me to restart the kernel.

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\Program Files\Spyder\pkgs\spyder\plugins\editor\plugin.py", line 2865, in stop_debugging
    ipyconsole.stop_debugging()
  File "C:\Program Files\Spyder\pkgs\spyder\plugins\ipythonconsole\plugin.py", line 749, in stop_debugging
    self.get_widget().stop_debugging()
  File "C:\Program Files\Spyder\pkgs\spyder\plugins\ipythonconsole\widgets\main_widget.py", line 2215, in stop_debugging
    sw.stop_debugging()
  File "C:\Program Files\Spyder\pkgs\spyder\plugins\ipythonconsole\widgets\debugging.py", line 391, in stop_debugging
    self.interrupt_kernel()
  File "C:\Program Files\Spyder\pkgs\spyder\plugins\ipythonconsole\widgets\shell.py", line 259, in interrupt_kernel
    super(ShellWidget, self).interrupt_kernel()
  File "C:\Program Files\Spyder\pkgs\qtconsole\frontend_widget.py", line 643, in interrupt_kernel
    self.kernel_manager.interrupt_kernel()
  File "C:\Program Files\Spyder\pkgs\jupyter_client\utils.py", line 30, in wrapped
    raise e
  File "C:\Program Files\Spyder\pkgs\jupyter_client\utils.py", line 27, in wrapped
    return loop.run_until_complete(future)
  File "C:\Program Files\Spyder\pkgs\nest_asyncio.py", line 89, in run_until_complete
    return f.result()
  File "asyncio\futures.py", line 178, in result
  File "asyncio\tasks.py", line 280, in __step
  File "C:\Program Files\Spyder\pkgs\jupyter_client\manager.py", line 599, in _async_interrupt_kernel
    raise RuntimeError("Cannot interrupt kernel. No kernel is running!")
RuntimeError: Cannot interrupt kernel. No kernel is running!
```

## Versions

* Spyder version: 5.3.3  (standalone)
* Python version: 3.8.10 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.7
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0                 :  1.4.1 (OK)
chardet >=2.0.0                      :  5.0.0 (OK)
cloudpickle >=0.5.0                  :  2.1.0 (OK)
cookiecutter >=1.6.0                 :  2.1.1 (OK)
diff_match_patch >=20181111          :  20200713 (OK)
intervaltree                         :  None (OK)
IPython >=7.31.1;<8.0.0              :  7.34.0 (OK)
jedi >=0.17.2;<0.19.0                :  0.18.1 (OK)
jellyfish >=0.7                      :  0.9.0 (OK)
jsonschema >=3.2.0                   :  4.14.0 (OK)
keyring >=17.0.0                     :  23.8.2 (OK)
nbconvert >=4.0                      :  7.0.0 (OK)
numpydoc >=0.6.0                     :  1.4.0 (OK)
paramiko >=2.4.0                     :  2.11.0 (OK)
parso >=0.7.0;<0.9.0                 :  0.8.3 (OK)
pexpect >=4.4.0                      :  4.8.0 (OK)
pickleshare >=0.4                    :  0.7.5 (OK)
psutil >=5.3                         :  5.9.1 (OK)
pygments >=2.0                       :  2.13.0 (OK)
pylint >=2.5.0;<3.0                  :  2.15.0 (OK)
pyls_spyder >=0.4.0                  :  0.4.0 (OK)
pylsp >=1.5.0;<1.6.0                 :  1.5.0 (OK)
pylsp_black >=1.2.0                  :  1.2.1 (OK)
qdarkstyle >=3.0.2;<3.1.0            :  3.0.3 (OK)
qstylizer >=0.1.10                   :  0.2.1 (OK)
qtawesome >=1.0.2                    :  1.1.1 (OK)
qtconsole >=5.3.2;<5.4.0             :  5.3.2 (OK)
qtpy >=2.1.0                         :  2.2.0 (OK)
rtree >=0.9.7                        :  1.0.0 (OK)
setuptools >=49.6.0                  :  65.3.0 (OK)
sphinx >=0.6.6                       :  5.1.1 (OK)
spyder_kernels >=2.3.3;<2.4.0        :  2.3.3 (OK)
textdistance >=4.2.0                 :  4.4.0 (OK)
three_merge >=0.1.1                  :  0.1.1 (OK)
watchdog                             :  2.1.9 (OK)
zmq >=22.1.0                         :  23.2.1 (OK)

# Optional:
cython >=0.21                        :  0.29.32 (OK)
matplotlib >=3.0.0                   :  3.5.3 (OK)
numpy >=1.7                          :  1.22.4 (OK)
pandas >=1.1.1                       :  1.4.3 (OK)
scipy >=0.17.0                       :  1.9.1 (OK)
sympy >=0.7.3                        :  1.11 (OK)

# Spyder plugins:
spyder_terminal.terminalplugin 1.2.2 :  1.2.2 (OK)
```

### Environment

<details><summary>Environment</summary>

```
# packages in environment at C:\Users\junea\miniconda3\envs\Base3.10:
#
# Name                    Version                   Build  Channel
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
branca                    0.5.0                    pypi_0    pypi
brotlipy                  0.7.0           py310he2412df_1004    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2022.9.24            h5b45459_0    conda-forge
certifi                   2022.9.24          pyhd8ed1ab_0    conda-forge
cffi                      1.15.1          py310hcbf9ad4_0    conda-forge
charset-normalizer        2.1.0              pyhd8ed1ab_0    conda-forge
cloudpickle               2.1.0              pyhd8ed1ab_0    conda-forge
colorama                  0.4.5              pyhd8ed1ab_0    conda-forge
conda-package-handling    1.8.1           py310h4f637d6_1    conda-forge
console_shortcut          0.1.1                         4  
cryptography              37.0.4          py310ha857299_0    conda-forge
dateutils                 0.6.12                     py_0    conda-forge
debugpy                   1.6.0           py310h8a704f9_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
folium                    0.12.1.post1             pypi_0    pypi
idna                      3.3                pyhd8ed1ab_0    conda-forge
intel-openmp              2022.1.0          h57928b3_3787    conda-forge
ipykernel                 6.15.1             pyh025b116_0    conda-forge
ipython                   7.33.0          py310h5588dad_0    conda-forge
jedi                      0.18.1          py310h5588dad_1    conda-forge
jinja2                    3.1.2                    pypi_0    pypi
jupyter_client            7.3.5              pyhd8ed1ab_0    conda-forge
jupyter_core              4.11.1          py310h5588dad_0    conda-forge
libblas                   3.9.0              15_win64_mkl    conda-forge
libcblas                  3.9.0              15_win64_mkl    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
liblapack                 3.9.0              15_win64_mkl    conda-forge
libsodium                 1.0.18               h8d14728_1    conda-forge
libzlib                   1.2.12               h8ffe710_2    conda-forge
markupsafe                2.1.1                    pypi_0    pypi
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
menuinst                  1.4.18          py310h5588dad_1    conda-forge
mkl                       2022.1.0           h6a75c08_874    conda-forge
nest-asyncio              1.5.5              pyhd8ed1ab_0    conda-forge
numpy                     1.23.1          py310h8a5b91a_0    conda-forge
openssl                   1.1.1q               h8ffe710_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pandas                    1.4.3           py310hf5e1058_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pip                       22.2.1             pyhd8ed1ab_0    conda-forge
powershell_shortcut       0.0.1                         3  
prompt-toolkit            3.0.30             pyha770c72_0    conda-forge
psutil                    5.9.1           py310he2412df_0    conda-forge
pycosat                   0.6.3           py310he2412df_1010    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pygments                  2.12.0             pyhd8ed1ab_0    conda-forge
pyopenssl                 22.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1           py310h5588dad_5    conda-forge
python                    3.10.5          h9a09f29_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python_abi                3.10                    2_cp310    conda-forge
pytz                      2022.1             pyhd8ed1ab_0    conda-forge
pywin32                   303             py310he2412df_0    conda-forge
pyzmq                     23.2.0          py310h73ada01_0    conda-forge
requests                  2.28.1             pyhd8ed1ab_0    conda-forge
ruamel_yaml               0.15.80         py310he2412df_1007    conda-forge
setuptools                63.2.0          py310h5588dad_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
spyder-kernels            2.3.2           py310h5588dad_0    conda-forge
sqlite                    3.39.2               h8ffe710_0    conda-forge
tbb                       2021.5.0             h2d74725_1    conda-forge
tk                        8.6.12               h8ffe710_0    conda-forge
tornado                   6.2             py310he2412df_0    conda-forge
tqdm                      4.64.0             pyhd8ed1ab_0    conda-forge
traitlets                 5.3.0              pyhd8ed1ab_0    conda-forge
tzdata                    2022a                h191b570_0    conda-forge
ucrt                      10.0.20348.0         h57928b3_0    conda-forge
urllib3                   1.26.11            pyhd8ed1ab_0    conda-forge
vc                        14.2                 hb210afc_6    conda-forge
vs2015_runtime            14.29.30037          h902a5da_6    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
win_inet_pton             1.1.0           py310h5588dad_4    conda-forge
xlsxwriter                3.0.3              pyhd8ed1ab_0    conda-forge
xz                        5.2.5                h62dcd97_1    conda-forge
yaml                      0.2.5                h8ffe710_2    conda-forge
zeromq                    4.3.4                h0e60522_1    conda-forge

```
</details>

**Comment by Pigankle** [comment]
I see here that it says "Cannot interrupt kernel...no kernel is running."  The last action I had tried was "Restart kernel."

**Comment by dalthviz** [comment]
Hi @Pigankle thank you for the feedback! As you mentioned I think this is is related/a duplicate of #19298 

Did you install Spyder for all the users or changed the default path for a current user installation? I think this error doesn't happpen with installations that use the current user option and don't change the defaul root path for the installation.

If you have any other info to help us understand what could be happening let us know!

**Comment by Pigankle** [comment]
> Hi @Pigankle thank you for the feedback! As you mentioned I think this is is related/a duplicate of #19298
> 
> Did you install Spyder for all the users or changed the default path for a current user installation? I think this error doesn't happpen with installations that use the current user option and don't change the default root path for the installation.
> 
> If you have any other info to help us understand what could be happening let us know!

I installed for all users with the installer.  I have since reinstalled with pip in powershell.  It is working as desired (fully functional 3.10 version), as long as I launch from powershell.  

I don't know if it is helpful, but I did find after I filed this bug report that there was a very reliable workaround:  Clicking on the little "X" to close the Console, rather than trying to use the restat or interrupt kernel commands from the menu.  Those sometimes (as in this bug report) would hang up, but closing the console pane didn't.  It would automatically reopen a new console, with the plaintext version of the error message ("Error when starting kernel: "The following argument was not expected: path/to/3.10installation) and then work fine as a python 3.10 interpreter.
